.gitleaks.toml

```diff
@@ -0,0 +1,5 @@
+[[allowlist]]
+description = "Placeholder API key in transitions.rs"
+commits = [
```

Comment:
The initial generation of a dummy `api-key` during commit [12faf7b](https://github.com/apollographql/rover/commit/12faf7bd764685b4327fa213c8e47092723f9f38) triggered the [Secrets in Source](https://github.com/apollographql/rover/runs/39821580829) failure in our CI/CD pipeline. We have removed this dummy key but now need to provide a flag to allow future tests to pass.